eng/scripts/Get-PkgVersion.ps1

```diff
@@ -7,26 +7,36 @@ param (
     [string] $PackageName
 )
 
-. ${PSScriptRoot}/SdkVersion-Common.ps1
+$repoRoot = Resolve-Path "$PSScriptRoot/../..";
+. ${repoRoot}\eng\common\scripts\logging.ps1
+. ${repoRoot}\eng\scripts\SdkVersion-Common.ps1
 
 $versionFileLocation = Get-VersionHppLocaiton `
```

Comment:
@danieljurek it looks like we have some room for some clean-up in the versioning scripts we use in cpp. We probably should move most of the logic here and in update into common functions in the shared common script. We should have it import the common.ps1 like the other languages. Lastly we probably should fix the typo in this function `Get-VersionHppLocation` :)

Reply:
Agreed on moving the common code between Get and Update package version scripts into the larger ecosystem of scripts. 

`Hpp` is not a typo. It's the extension we're using for C++ header files which contains the explicit version constants ([example](https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/core/azure-core/inc/azure/core/version.hpp))

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/1764

Reply:
Hpp isn't the typo it is the `Locaition` vs `Location`